app/jobs/delete_old_zendesk_tickets_job.rb

```diff
@@ -2,11 +2,7 @@ class DeleteOldZendeskTicketsJob < ApplicationJob
   def perform
     tickets = ZendeskService.find_closed_tickets_from_6_months_ago
     return if tickets.size.zero?
-    if tickets.size >= 100
-      raise "More than 100 tickets to delete were found in Zendesk. " \
-              "The DeleteOldZendeskTicketsJob doesn't handle pagination. " \
-              "If you're seeing this, we need to build pagination."
-    end
+    DeleteOldZendeskTicketsJob.perform_later if tickets.size >= 100
```

Comment:
That's one way to do it @felixclack but running the job recursively should also work. Each run will attempt to delete the first 100 tickets it finds, and it will stop recursing when it gets to the last job (<100 tickets).

The downside to paginating currently is that it will probably fall afoul of the rate limits. Zendesk's API has a 400 request rate limit and we run a `.group.name` call for each ticket when we build up the `ZendeskDeleteRequest`s below. I'm trying to solve that separately but in case it doesn't work, batch deleting 100 at a time with a timeout between will work for sure.